libproxy/modules/config_w32reg.cpp

```diff
@@ -160,7 +161,10 @@ class w32reg_config_extension : public config_extension {
 		if (get_registry(W32REG_BASEKEY, "ProxyOverride", &tmp, NULL, NULL)) {
 			string po = tmp;
 			delete tmp;
-			if (po == "<local>")
+			const char windowsDelimiter = ';';
+			const char libproxyDelimiter = ',';
+			replace(po.begin(), po.end(), windowsDelimiter, libproxyDelimiter );
```

Comment:
Do we need to worry about semicolons that may be escaped for any reason? Are semicolons allowed here outside of being a delimieter?

Reply:
Short answer, no we do not need to worry about escaped semi-colons in the proxy bypass list since windows would otherwise not support these. The values get read from a Registry key and are set here: 
![image](https://user-images.githubusercontent.com/3265891/92147848-aa630380-ede9-11ea-99b6-1f76c3a07248.png)

Reply:
That makes sense. I didn't think semicolons would be allowed in an address, but I wasn't positive on the entire schema of it all. I did try entering some semicolons into that address list UI and it stripped out any repeat semicolons I had, so it does seem like it's just assuming semicolons are always a delimeter and never part of hte address.